app/views/unwithdraw/confirm.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, t("unwithdraw.confirm.title", title: @edition.title_or_fallback) %>
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <p class="govuk-body">
+          <%= render_govspeak(t("unwithdraw.confirm.description_govspeak")) %>
+        </p>
```

Comment:
This will be producing invalid HTML as the govspeak will be generating paragraph elements and they're not allowed to be nested within a paragraph element.

You should embed within a `div` rather than p for this.

Reply:
Just noticed that we've got this on the previous PR too btw.

Reply:
Thanks for this - my lack of understanding of how govspeak works.  Will change.

Reply:
Fixed